app/assets/javascripts/template_invocation.js

```diff
@@ -96,7 +96,7 @@ function delayed_refresh(url, data){
         url: url,
         data: data,
         dataType: "script",
-        error: function() { $("div.terminal div.printable").append(__('<div class="line error">There was an error while updating the status, try <a href="javascript:window.location.href=window.location.href">refreshing</a> the page</div>')) }
+        error: function() { $("div.terminal div.printable").append(__('<div class="line error">There was an error while updating the status, try refreshing the page.</div>')) }
```

Comment:
HTML should be outside of the translated string, concatenate them together.

I'm not actually sure that Javascript translation is in place for plugins anyway, I think you'll find that it's always English as there's no mechanism to generate and include app/assets/javascripts/locale/ files like Foreman core does.